.github/ISSUE_TEMPLATE/beta_feature.md

```diff
@@ -3,35 +3,72 @@ name: Beta Feature Requirements
 about: Requirements for promoting a feature to beta
 ---
 
-# Beta Feature Requirements
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the issue author with submission of an issue)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
 
-This page lists the requirements for promoting a feature to beta. Promotion to beta must meet all requirements for promotion to alpha. Please check off and document the steps as they are completed.
+[//]: # (This page lists the requirements for an experimental feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the issue.)
+
+# Feature: 
+
+[//]: # (All information in this section is mandatory for new issue submission.)
+
+**Feature Name:**  
+
+[//]: # (The name of the feature, e.g. Multiple control planes)
+
+**Primary Lead(s):**
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+**Short description:**
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
 
-**Feature Name:** 
+**RFC:**
 
-**Design Doc:**
+[//]: # (This outlines the ideas for a feature and is used to collect early feedback. See the RFC template here: https://docs.google.com/document/d/1ewJoCcw5-04crH-M0xw4zFxz1cfwVCPnNyW4K3m4Yyc/template/preview)
 
-**Alpha Checklist:**
+**Design doc:**
+
+[//]: # (This is used for discussion of design and testing for a feature. See the design doc template here:https://docs.google.com/document/d/16FLQK8uhhic1ovKnnOG3OXJjFKs2aHnSmbximidpKwM/template/preview)
```

Comment:
this type of comment may confuse the rendering - I can't remember if I was able o render HTTP correctly vs not have it rendered when the issue was created. Can you verify?